docs/iot-gateway/configuration.md

```diff
@@ -15,7 +15,7 @@ description: Installation structure and configuration of ThingsBoard IoT Gateway
 Please see default directory structure below for daemon installation.  
 
 ```text
-/etc/thingsboard-gateway/config                   - Configuration folder.
+/thingsboard-gateway/config                   - Configuration folder.
```

Comment:
In case of using deb/rpm package to install - we have another address for configuration folder, like it was earlier.